tests/unit/test_event_emitter.cpp

```diff
@@ -26,12 +26,12 @@
  *    POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <CppUTest/TestHarness.h>
-
 #include <stdarg.h>
 
 #include "common/event_emitter.hpp"
 
+#include <CppUTest/TestHarness.h>
```

Comment:
This line is moved intentionally to fix a build error, see https://stackoverflow.com/questions/54611227/compilation-error-after-including-jsoncpp